LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-anchoring/shadow-dom-subscroller-expected.txt

```diff
@@ -0,0 +1,5 @@
+Scrolling over this element doesn't scroller the overflow scroller
```

Comment:
-expected.txt needs updating with the new sentence

Reply:
Fixed.